### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Splend1ed**

### CODE REVIEW

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_cat_and_god_years_convert():
```
Comment (IvanRamyk) [inline app/test_main.py:4]: You should provide more tests for boundary values and classes of equivalence

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

File: app/test_main.py [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+                15, 15,
+                [1, 1],
+                id="should return 1 human year for first animal years"
+            ),
+            pytest.param(
+                24, 24,
+                [2, 2],
+                id="should add `extra year` every 9 animal next years"
+            ),
+            pytest.param(
```
Comment (IvanRamyk) [inline app/test_main.py:19]: Could you add a test [28, 28] to make sure that the numbers of extra years are really different

File: app/test_main.py [hunk 3]
```diff
@@ -1,3 +1,27 @@
...
+class TestCatDogYears:
+    @pytest.mark.parametrize(
+        "cat_years, dog_years, human_years",
+        [
+            pytest.param(
+                15, 15,
+                [1, 1],
+                id="should return 1 human year for first animal years"
+            ),
+            pytest.param(
```
Comment (IvanRamyk) [inline app/test_main.py:14]: Could you add a test [23, 23] to make sure that the function uses exactly a 9-year difference?

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

**Review by IvanRamyk — APPROVED** [review]
Good job!
